src/App.js

```diff
@@ -18,7 +23,14 @@ const App = () => {
   return (
     <div className={wrapper}>
       <SiteHeading />
-      <MarketPage />
+      <Switch>
+        <Route exact path="/">
+          <Redirect to={`/exchange/${DEFAULT_CRYPTO}`} />
+        </Route>
+        <Route exact path="/exchange/:currency?">
```

Comment:
This `exact` can be a problem when we would like to display wallet/transaction modals as a subroute of that one.